modules/api/test/ApiTest.php

```diff
@@ -0,0 +1,392 @@
+<?php declare(strict_types=1);
+/**
+ * This contains tests relevant to the API
+ *
+ * PHP Version 7
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+namespace LORIS\api\Test;
+ini_set('memory_limit', '1024M');
+
+require './vendor/autoload.php';
+require_once __DIR__ .
+    "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+
+use phpDocumentor\Reflection\Types\AbstractList;
+use phpDocumentor\Reflection\Types\Array_;
+use PHPUnit\Framework\TestCase;
+use Laminas\Diactoros\ServerRequest;
+use GuzzleHttp\Client;
+
+/**
+ * PHPUnit class for API testsuite
+ *
+ * @category   API
+ * @package    Tests
+ * @subpackage Login
+ * @author     Simon Pelletier <simon.pelletier@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+
+class ProjectsTest extends TestCase
+{
+
+    private $_client;
+    /**
+     * @var string[]
+     */
+    private $_headers;
+    private $_base_uri;
+
+    /**
+     * @var string[]
+     */
+
+    private function _login()
+    {
+        $this->_base_uri = 'https://test-loris-dev.loris.ca/api/v0.0.3/';
+        $this->_client = new \GuzzleHttp\Client(['base_uri' => $this->_base_uri]);
+        $response = $this->_client->request('POST', 'https://test-loris-dev.loris.ca/api/v0.0.3/login', ['json' => ['username' => 'admin', 'password' => 'LORISitb100%']]);
```

Comment:
```suggestion
        $response = $this->_client->request('POST', 'https://test-loris-dev.loris.ca/api/v0.0.3/login', ['json' => ['username' => 'admin', 'password' => 'LORISitb100%']]);
```

I don't think we want the password to this VM to be publicly exposed. As @kongtiaowang pointed out it's not necessarily stable. 

I'm trying to think whether there's an easy solution here. If there is a way to do this within the server Travis creates instead, that would be good. If not then it might be worth making a point in the LORIS meeting about how to approach API testing and what server to test on.

Reply:
The easist way is that using integration test on Travis directly.
https://web:8000/api/v0.0.3/...